Sources/Sentry/SentrySDK.m

```diff
@@ -385,7 +385,10 @@ + (void)captureEnvelope:(SentryEnvelope *)envelope
 + (void)storeEnvelope:(SentryEnvelope *)envelope
 {
     if (nil != [SentrySDK.currentHub getClient]) {
-        [[SentrySDK.currentHub getClient] storeEnvelope:envelope];
+        [[SentrySDK.currentHub getClient]
```

Comment:
`m`: What about adding a method `storeEnvelope` to the hub, which calls the `storeEnvelope` on the client and updates the session state? Then we don't need to expose the `updateSessionState` and we have the session logic encapsulated in the hub.